tests/test_memory_utils.py

```diff
@@ -108,7 +122,14 @@ def mock_training_loop_function(batch_size):
     @require_non_torch_xla
     def test_release_memory(self):
         starting_memory = memory_allocated_func()
-        model = ModelForTest()
+
+        if torch_device.startswith("hpu"):
+            # hpu has a minimum memory allocation that cannot be released,
```

Comment:
Wow TIL

Reply:
yeah it was fun digging why the test fails. right from the moment you `import habana_frameworks.torch`, you get `torch.hpu.memory_allocated()` ~ 5 MB.